actioncable/lib/assets/javascripts/action_cable/connection.coffee

```diff
@@ -75,7 +79,17 @@ class ActionCable.Connection
     error: ->
       @disconnect()
 
+    dispatchEvent: (identifier, type, message) ->
+      unless identifier == "_ping"
+        event = document.createEvent('Event')
+        event.initEvent("cable:#{type}", true, false)
+        channel = JSON.parse(identifier).channel
+        event.cable = {data: message, identifier: channel}
```

Comment:
I'm -1 on using `event.cable` as the property and would prefer an `event.data` object like  `{ channel: channel, message: message }`.